inlong-sdk/sort-sdk/src/test/java/org/apache/inlong/sdk/sort/manager/InlongSingleTopicManagerTest.java

```diff
@@ -0,0 +1,160 @@
+
+        InLongTopicFetcher inLongTopicFetcher = inLongTopicManager.addFetcher(inLongTopic);
+        Assert.assertNull(inLongTopicFetcher);
+    }
+
+    @Test
+    public void testRemoveFetcher() {
+
+        InLongTopicFetcher inLongTopicFetcher = inLongTopicManager.removeFetcher(inLongTopic, true);
+        Assert.assertNull(inLongTopicFetcher);
+
+        ConcurrentHashMap<String, InLongTopicFetcher> fetchers = new ConcurrentHashMap<>();
+        InLongTopicFetcher inLongTopicFetcherRmMock = PowerMockito.mock(InLongTopicFetcher.class);
+        fetchers.put(inLongTopic.getTopicKey(), inLongTopicFetcherRmMock);
+
+        Whitebox.setInternalState(inLongTopicManager, "fetchers", fetchers);
+
+        inLongTopicFetcher = inLongTopicManager.removeFetcher(inLongTopic, true);
+        Assert.assertNotNull(inLongTopicFetcher);
+
+    }
+
+    @Test
+    public void testGetFetcher() {
+        InLongTopicFetcher fetcher = inLongTopicManager.getFetcher(inLongTopic.getTopicKey());
+        Assert.assertNull(fetcher);
+        ConcurrentHashMap<String, InLongTopicFetcher> fetchers = new ConcurrentHashMap<>();
+        InLongTopicFetcher inLongTopicFetcherRmMock = PowerMockito.mock(InLongTopicFetcher.class);
+        fetchers.put(inLongTopic.getTopicKey(), inLongTopicFetcherRmMock);
+
+        Whitebox.setInternalState(inLongTopicManager, "fetchers", fetchers);
+
+        fetcher = inLongTopicManager.getFetcher(inLongTopic.getTopicKey());
+        Assert.assertNotNull(fetcher);
+
+    }
+
+    @Test
+    public void testGetManagedInLongTopics() {
+        Set<String> managedInLongTopics = inLongTopicManager.getManagedInLongTopics();
+        Assert.assertEquals(0, managedInLongTopics.size());
+
+        ConcurrentHashMap<String, InLongTopicFetcher> fetchers = new ConcurrentHashMap<>();
+        InLongTopicFetcher inLongTopicFetcherRmMock = PowerMockito.mock(InLongTopicFetcher.class);
+        fetchers.put(inLongTopic.getTopicKey(), inLongTopicFetcherRmMock);
+        Whitebox.setInternalState(inLongTopicManager, "fetchers", fetchers);
+        managedInLongTopics = inLongTopicManager.getManagedInLongTopics();
+        Assert.assertEquals(1, managedInLongTopics.size());
+
+    }
+
+    @Test
+    public void testGetAllFetchers() {
+        Collection<InLongTopicFetcher> allFetchers = inLongTopicManager.getAllFetchers();
+        Assert.assertEquals(0, allFetchers.size());
+
+        ConcurrentHashMap<String, InLongTopicFetcher> fetchers = new ConcurrentHashMap<>();
+        InLongTopicFetcher inLongTopicFetcherRmMock = PowerMockito.mock(InLongTopicFetcher.class);
+        fetchers.put(inLongTopic.getTopicKey(), inLongTopicFetcherRmMock);
+        Whitebox.setInternalState(inLongTopicManager, "fetchers", fetchers);
+        allFetchers = inLongTopicManager.getAllFetchers();
+        Assert.assertEquals(1, allFetchers.size());
+    }
+
+    @Test
+    public void testOfflineAllTp() {
```

Comment:
What does the `Tp` mean? 
If it is `topic`, then using the `Topics` is better.

Reply:
it means topic and partition